ts/utils/util.py

```diff
@@ -81,11 +96,14 @@ def map_class_to_label(probs, mapping=None, lbl_classes=None):
 
     return results
 
+
 class PredictionException(Exception):
     def __init__(self, message, error_code=500):
         self.message = message
         self.error_code = error_code
         super().__init__(message)
 
     def __str__(self):
-        return "message : error_code".format(message=self.message, error_code=self.error_code)
+        return "{message} : {error_code}".format(
```

Comment:
since we're here do you mind changing this to an `f` string

Reply:
yeah sure, I was wondering whether you are being conservative for earlier python versions, otherwise I find f-strings much more concise. Does this mean that, I should replace `%s` style substitutions with f-strings if I hit them?

Reply:
Yeah for security reasons we support only Python 3.8+. I've been wanting to do all these upgrades but it's easier to manage if we do it per new file that someone happened to touch